osx/main.py

```diff
@@ -0,0 +1,104 @@
+import json
+import urllib2
+from os.path import expanduser
+
+ENVZ = 'envz'
+SVCZ = 'svcz'
+NOOP = 'noop'
+required_keys = [ENVZ, SVCZ]
+super_secret_proxy_tmpl = "http://status.bign8.info/proxy?url={}"
+widget_icon = u"\u03F2".encode('utf-8')
```

Comment:
TODO - it would be cool to make this red if all service checks are < 300, else red to mimic what the dart app does already